static/css/article.css

```diff
@@ -26,6 +26,17 @@ body {
   margin-top: 2rem;
 }
 
+.blog-article .content a {
+  color: var(--color-primary);
```

Comment:
Doesn't `color: inherit;` work?

Reply:
Nope. I guess it's because color is not defined for `a`.